.github/workflows/release.yaml

```diff
@@ -1,23 +1,31 @@
 name: "release"
 
 on:
-  push:
-    tags:
-      - "release/*"
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (E.g. M4 or M4a)'
+        required: true
+        type: string
+      target:
+        description: 'Ref to use for this release, defaults to trunk'
+        required: true
+        default: 'trunk'
+        type: string
 
 jobs:
   release:
     name: "create_release"
-    runs-on: "ubuntu-latest"
+    runs-on: ubuntu-22.04
```

Comment:
Weirdly this is newer than `ubuntu-latest` which is at ubuntu-20;

I need it because I rely on the most recent version of gh.